pkgs/development/haskell-modules/configuration-common.nix

```
@@ -3218,6 +3218,19 @@ with haskellLib;
   # and therefore aren't uploaded to hackage
   # Needs to be fixed upstream
   haskore = dontCheck (doJailbreak super.haskore);
+
+  # 2025-07-10: Hackage release is outdated, https://github.com/portnov/libssh2-hs/issues/77
+  libssh2 = overrideSrc {
+    version = "unstable-2025-04-03";
```

`pkgs/README.md` prescribes

```suggestion
    version = "0.2.0.9-unstable-2025-04-03";
```

now.